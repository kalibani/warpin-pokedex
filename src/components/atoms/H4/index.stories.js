// Stories H4 Component
// --------------------------------------------------------
import React, { Fragment } from 'react';
import { storiesOf } from '@storybook/react';
import H4 from './index';
import Notes from './readme.md';

storiesOf('Atoms', module).add('H4', () => (
  <H4 />
), {
  info: { inline: true, header: false, text: Notes }
});
