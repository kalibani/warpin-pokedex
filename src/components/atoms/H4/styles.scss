/* ==========================================================================
Styles for H4 component
========================================================================== */

// import mixin and variables
@import "../../../assets/styles/variables";

// Start styles H4 component
.a-h4 {
  color: $outer-space;
  font-size: 20px;
  font-weight: 600;

}
