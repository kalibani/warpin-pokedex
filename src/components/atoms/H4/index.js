// H4 Component
// --------------------------------------------------------

import React from 'react';
import PropTypes from 'prop-types';
import './styles.scss';

const H4 = ({ propsName }) => (
  <tag>
    Functional Component (Stateless Component)
  </tag>
);

H4.propTypes = {
  propsName: PropTypes.string
};

H4.defaultProps = {
  propsName: ''
};

export default H4;
