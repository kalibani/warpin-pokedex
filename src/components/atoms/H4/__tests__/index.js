// Test H4 Component
// --------------------------------------------------------

import React from 'react';
import H4 from '../../index';
import renderer from 'react-test-renderer';

test('H4 is match the snapshot', () => {
const component = renderer.create(
H4,
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});
