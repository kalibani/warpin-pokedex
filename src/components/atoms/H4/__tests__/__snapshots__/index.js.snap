// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H4 is match the snapshot 1`] = `null`;
